Packages/tlp.udonvoiceutils/Runtime/Examples/VoiceOverrideTriggerZone.cs

```diff
@@ -23,13 +23,17 @@ public class VoiceOverrideTriggerZone : TlpBaseBehaviour
 
         private Collider[] _allTrigger;
 
+        private int[] playerIds = new int[82];
+        private int[] colliderStacks = new int[82];
```

Comment:
Please use `DataList` or better a `DataDictionary` instead of arrays, key is playerid, value is stacksize.